gridded_etl_tools/utils/store.py

```diff
@@ -420,7 +419,10 @@ def path(self) -> pathlib.Path:
         pathlib.Path
             Path to the Zarr on the local filesystem
         """
-        return self.dm.output_path().joinpath(f"{self.dm.name()}.zarr")
+        if self.dm.custom_output_path:
+            return self.dm.custom_output_path
```

Comment:
Currently, in Convenience.output_path, custom_path is not expected to contain a filename. It's just the directory where the data will be written, and the filename is generated by the class. That's probably how it should be treated by the store module as well, at least until we do a general clean up of input/output paths.

Reply:
I don’t like this approach. During testing — the only time we normally use this flag — I find it useful to output files in the anticipated directory with different file names indicating key aspects of the test. For S3 in particular it would be unhelpful to have to create a new bucket/directory for each output.

Reply:
That's fair, but it would be bad for two functions to be expecting different things from the same parameter. I took a quick look at tools and managers and I'm not sure anything is using Convenience.output_path besides the convenience module itself and the one line in this store module, so maybe we could just erase Convenience.output_path and define custom_output_path to be a path to a Zarr.

Reply:
The only place I can see using it is `create_output_path` so I’ll rewrite that.

Reply:
I think that function can just be erased too

Reply:
Well we need it to automatically create the output path for Zarrs on the local file system, no?

Reply:
I'm just going by looking at a grep of tools and managers code and I don't see anything using create_output_path

But also I just realized a Zarr is a directory so maybe it doesn't even matter

Reply:
Might be worth cleaning out those functions though

Reply:
Wild. I’m down to drop `create_output_path` if it’s not being used.

Reply:
writing to zarr uses the store module now for all stores so those convenience functions may not be used anymore